pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -1,5 +1,9 @@
-.dance-subheading
-  %a{href: "https://code.org/translate/dance"}= I18n.t(:hoc2018_dance_party_subheading)
+%div.dance-text-icon-container
```

Comment:
I still think conceptually this subheading is better suited as part of `dance.haml` since it's not specific to the tutorial and you won't want it in the partial when you generalize to use across starwars/minecraft/dance.